### PR TITLE
Fixing example to be more complete

### PR DESCRIPTION
**Pull request by andyfleming**
(to function when copy-pasted)

### CODE REVIEW

File: syntax_and_semantics/methods_and_instance_variables.md [hunk 1]
```diff
@@ -7,13 +7,25 @@ class Person
   def initialize(@name : String)
     @age = 0
   end
+  
+  def age
```
Comment (mverzilli) [inline syntax_and_semantics/methods_and_instance_variables.md]: Maybe we should also write the `name` getter, what do you think? Mostly to comply with "ask for its name and for its age" in the paragraph below

File: syntax_and_semantics/methods_and_instance_variables.md [hunk 2]
```diff
@@ -7,13 +7,25 @@ class Person
...
 
 Right now, we can't do much with a person: create it with a name, ask for its name and for its age, which will always be zero. So lets add a method that makes a person become older:
 
 ```crystal
 class Person
+  def initialize(@name : String)
+    @age = 0
+  end
+  
+  def age
```
Comment (mverzilli) [inline syntax_and_semantics/methods_and_instance_variables.md]: Ditto.

File: syntax_and_semantics/methods_and_instance_variables.md (hunk 1 above)
Comment (karlseguin) [inline syntax_and_semantics/methods_and_instance_variables.md]: I think the original commit is good. The phrase that you quote appears to have been removed by https://github.com/crystal-lang/crystal-book/commit/8b493993f435c9e04658783751faa01f5b5c29c4

**Comment by mverzilli** [comment]
Thank you!

**Pull request merged**